src/cljs/bluegenes/pages/results/events.cljs

```diff
@@ -163,3 +164,43 @@
                         :enrichment-results
                         (keyword (:widget params))] nil)
       :enrichment/get-enrichment [(:widget params) enrichment-chan]})))
+
+(reg-event-db
+ :description/edit
+ (fn [db [_ state]]
+   (-> db
+       (assoc-in [:results :description :editing?] state)
+       (assoc-in [:results :errors :description] false))))
+
+(reg-event-fx
+ :description/update
```

Comment:
as above - given this page can be used for non-list queries, all list update/edit functions should probably mention that they're list-specific functions.